example/follow_me/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+cmake_minimum_required(VERSION 2.8.12)
+
+project(follow_target)
```

Comment:
CMakeLists.txt for examples get a lot easier once this is in: https://github.com/dronecore/DroneCore/pull/130/files#diff-b33c729b1001aff667a2a60dcf6be327R6.